### ISSUE TITLE
Support Swift 4.0, Xcode 9

### ISSUE THREAD

**Issue by dovanvu1792** [loregr/LGButton#19]
Hello,
Have some errors when compiler with swift 4.0.

**Comment by annjawn** [comment]
So rectifying the code using Xcode autocorrection worked for the most part in resolving all the errors. Though it shows a few warning at a couple of places about `String.characters` being deprecated in Swift 4, but it still works. One more thing I had to do is to override `prepareForInterfaceBuilder()` in the `LGButton.swift` class file in order for the IB to start showing the changes made in the Attributes inspector at build time. It's weird because this never happened for LGButton on Swift 3 Xcode 8.

```
override public func prepareForInterfaceBuilder() {
        super.prepareForInterfaceBuilder()
}
```

But for now it works just fine.

**Comment by loregr** [comment]
Implemented in [v1.0.4](https://github.com/loregr/LGButton/releases/tag/1.0.4)
